app/uk/gov/hmrc/tradergoodsprofilesrouter/controllers/GetRecordsController.scala

```diff
@@ -48,6 +48,20 @@ class GetRecordsController @Inject() (
     result.merge
   }
 
+  def getTGPRecords(
```

Comment:
This is only referenced by the app.routes (as apposed to getTGPRecord which is used by GetRecordsControllerSpec) does this mean it's not being tested?

Reply:
added the missing unit test for controller, thanks Ben